File: C/1038.c
#include <stdio.h>

float main(){
	float total;
	int codigo, quantidade;
	scanf("%d%d", &codigo, &quantidade);
	if (codigo==1){
		total = quantidade * 4.0;
	}
	if (codigo==2){
		total = quantidade * 4.5;
	}
	if (codigo==3){
		total = quantidade * 5.0;
	}
	if (codigo==4){
		total = quantidade * 2.0;
	}
	if (codigo==5){
		total = quantidade * 1.5;
	}
	printf("Total: R$ %.2f\n", total);
	return 0;
}

File: Python/1038.py
item, quantidade = [float(x) for x in input().split()]
if(item == 1):
    valor = quantidade*4.0
if(item == 2):
    valor = quantidade*4.5
if(item == 3):
    valor = quantidade*5.0
if(item == 4):
    valor = quantidade*2.0
if(item == 5):
    valor = quantidade*1.5
print("Total: R$ {:.2f}".format(valor))